pyinfra/api/connectors/util.py

```diff
@@ -240,7 +240,7 @@ def make_unix_command(
     command,
     env=None,
     chdir=None,
-    shell_executable='sh',
+    shell_executable=None,
```

Comment:
I think this should remain `'sh'` for now as the default and `None`/no shell has to be explicitly set.

Reply:
Okay. Fixing.